.github/workflows/ets-from-source.yml

```diff
@@ -0,0 +1,52 @@
+name: Test with EDM using ETS packages from source
+
+on:
```

Comment:
I suggest also adding `workflow_dispatch:` as a trigger, so that we can run this job manually.